### ISSUE TITLE
Stagger animations no longer work on Chrome

### ISSUE THREAD

**Issue by GFoley83** [nozzle/nzAnimate#11]
Stagger animations no longer work on Chrome.
Tested at the demo page: http://nozzle.github.io/nzAnimate/
On Chrome 42 for Windows 8.1 and OS X Yosemite.

**Comment by tannerlinsley** [comment]
It looks like the ng-EVENT-stagger classes aren't even being added to the repeat elements.  Can you confirm?

**Comment by GFoley83** [comment]
@tannerlinsley Yep, can confirm. The `.ng-enter-stagger` and`.ng-leave-stagger` classes are not getting added any more.

**Comment by tannerlinsley** [comment]
That is what's breaking it. Could be something wrong with the angular
version I'm using.

**Comment by GFoley83** [comment]
Submitted a pull request with a fix for the broken stagger animation: #15
